packages/katello/katello-repos/katello-repos.spec

```
@@ -17,11 +17,12 @@ Group:          Applications/Internet
 License:        GPLv2
 URL:            https://theforeman.org/plugins/katello/
 Source0:        katello.repo
-Source1:        RPM-GPG-KEY-katello-2015
 
 BuildArch:      noarch
 
-BuildRequires: sed
+BuildRequires:  sed
+
+Requires:       foreman-release
```

Do we want to enforce a match here?
```suggestion
Requires:       foreman-release >= 1.24
Requires:       foreman-release < 1.25
```

---

That would require updating the lock on every major release. I'd defer that until we merge the two packages? :)

---

I guess that's fine. Users will find out it's incompatible anyway when the GPG signatures don't match ;)